src/FAI3_backend/src/config_management.rs

```diff
@@ -35,3 +35,20 @@ pub fn get_config(config_key: String) -> Result<String, GenericError> {
         }
     }
 }
+
+pub fn internal_get_config(config_key: String) -> Result<String, GenericError> {
```

Comment:
The `internal_*` methods are used because the timer queue is not admin, so we need these method alternatives to call it when running things from the job queue without requiring admin rights.